skbeam/core/accumulators/projector.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+
+"""
+authors: Mikhail Dubrovin, TJ Lane, Christopher O'Grady
+"""
+
+
+class Projector(object):
```

Comment:
@tacaswell Howzabout BinImage as a name?  (and RadialBinImage for the specific radial case).  "Bin" could conceivably be confused with "binary" but I think most photon science users would not do that.

One thing I do need your input on:  what directory should this go in?